common/doc-dev/conf.py

```diff
@@ -61,7 +61,7 @@
 author = 'Germar Reitze'
 
 # Don't edit this variable. It is updated automatically by "updateversion.sh".
-version = '1.4.4-dev'
+version = '1.4.4-dev.bb32455a'
```

Comment:
I am not sure if we should merge "hashed" version numbers or not (may not harm but also be misleading if we don't `make`/`updateversion` before each push

Reply:
Ah, yes. I did a `make install` and forget that.
Mhm... I could add a unit test that run on Travis only checking that.